contents/docs/product-analytics/troubleshooting.mdx

```diff
@@ -0,0 +1,109 @@
+---
+title: Troubleshooting and FAQs
+---
+
+Having trouble with events? There are a few common reasons that you may not see events appear in your project.
+
+### 1. The library is being used incorrectly. (this is usually the issue)
+
+> Warning: Make sure to not use sensitive data while debugging using third-party tools.
+
+1. Visit [webhook.site](https://webhook.site/) and copy “Your unique URL”.
+2. Inside your code replace [posthog.com](https://app.posthog.com) (or [eu.posthog.com](https://eu.posthog.com) if using the EU Cloud) with “Your unique URL”.
+3. Make sure that the SDK sends a couple of events.
+4. See if the incoming events arrive as expected.
+
+![Events visible on webhook.site](../../images/docs/product-analytics/events-in-webhook.site.png)
+
+### 2. There is a bug within the library being used.
+
+#### Front-end libraries
+1. Open the devtools and switch to the network tab.
```

Comment:
This is only for the JavaScript web library right? (Front-end technically includes iOS, Android, React Native, and Flutter)

Reply:
right, ios, android, react native, and flutter should use the wireshark approach
let me correct that

Reply:
changed wording to web / other